### PR TITLE
VMTensor .At() and .setAt() methods refactoring

### PR DESCRIPTION
**Pull request by vhavrylov-ncube**

### CODE REVIEW

File: libs/vm-modules/src/math/tensor.cpp [hunk 1]
```diff
@@ -57,19 +57,20 @@ Ptr<VMTensor> VMTensor::Constructor(VM *vm, TypeId type_id, Ptr<Array<SizeType>>
...
       })
-      .CreateMemberFunction("at", &VMTensor::AtOne)
-      .CreateMemberFunction("at", &VMTensor::AtTwo)
-      .CreateMemberFunction("at", &VMTensor::AtThree)
-      .CreateMemberFunction("at", &VMTensor::AtFour)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtOne)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtTwo)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtThree)
-      .CreateMemberFunction("setAt", &VMTensor::SetAtFour)
+      .CreateMemberFunction("at", &VMTensor::At<Index>)
```
Comment (baykaner) [inline libs/vm-modules/src/math/tensor.cpp:66]: it's annoying but there's no obvious way around this right now is there?
If so, would you mind please adding in the bindings for the 5 & 6 dimensional tensor accessor/setter as well. There are some cases with convolution where you might get such a tensor.

Comment (baykaner) [reply libs/vm-modules/src/math/tensor.cpp:66]: you can do a separate pr for that if you like

**Review by baykaner — DISMISSED** [review]

File: libs/vm-modules/src/math/tensor.cpp (hunk 1 above)
Comment (vhavrylov-ncube) [reply libs/vm-modules/src/math/tensor.cpp:66]: Done

**Review by baykaner — APPROVED** [review]
👍

**Pull request merged**